docs/docs/how-to/disable-api-database.md

```diff
@@ -26,7 +36,7 @@ Open up `netlify.toml`. We're going to comment out one line:
 
 ```toml {4}
 [build]
-  command = "yarn rw build"
+  command = "yarn rw deploy netlify --prisma=false --dm=false"
```

Comment:
@jtoar How might we not build or deploy the api on non-Netlify deploys?

Reply:
It looks like for vercel and layer0, it'd be the same. For render, it may be as easy as deleting the config in the `render.yaml` file. Same for flightcontrol, but in `flightcontrol.json`. And for serverless I'm not sure. That one may just work if you delete the api side since the config is there as well.